include/zephyr/bluetooth/gatt.h

```diff
@@ -167,16 +167,57 @@ typedef ssize_t (*bt_gatt_attr_write_func_t)(struct bt_conn *conn,
 
 /** @brief GATT Attribute structure. */
 struct bt_gatt_attr {
-	/** Attribute UUID */
+	/** @brief Attribute Type, aka. "UUID" */
 	const struct bt_uuid *uuid;
-	/** Attribute read callback */
+
+	/** @brief Attribute Value function
+	 *
+	 * This is the general API to read from the Attribute Value of
+	 * this Attribute in on-air format.
+	 *
+	 * This function may safely assume the Attribute Permissions are
+	 * satisfied for this read.
+	 *
+	 * This function is primarily invoked by the GATT server when a
+	 * remote GATT client performs a read, but it can also be
+	 * invoked for a local read.
+	 *
+	 * May be NULL if the attribute is not readable.
+	 */
 	bt_gatt_attr_read_func_t read;
-	/** Attribute write callback */
+
+	/** @brief Attribute Value write method
```

Comment:
```suggestion
	/** @brief Attribute Write Value function
```
Suggest to add "Read" and "Write" before "Value", but I'm fine with putting it after "Value" as well. We should use either "method" or function", but not both.

Reply:
Ok. I'll make it "Attribute Value write method".